test/server/listener_manager_impl_quic_only_test.cc

```diff
@@ -49,13 +49,22 @@ reuse_port: true
 
   envoy::config::listener::v3::Listener listener_proto = parseListenerFromV2Yaml(yaml);
   EXPECT_CALL(server_.random_, uuid());
-  expectCreateListenSocket(envoy::config::core::v3::SocketOption::STATE_PREBIND,
+  expectCreateListenSocket(
+      envoy::config::core::v3::SocketOption::STATE_PREBIND,
 #ifdef SO_RXQ_OVFL
-                           /* expected_num_options */ 3, // SO_REUSEPORT is on as configured
+#ifdef UDP_GRO
+      /* expected_num_options */ 4, // SO_REUSEPORT and UDP_GRO is on as configured
 #else
-                           /* expected_num_options */ 2,
+      /* expected_num_options */ 3, // SO_REUSEPORT is on as configured
 #endif
```

Comment:
Change this bulk to `(supportUdpGro() ? 4 : 3)`

Reply:
Changed this to 4 here, since either UdpGro or and empty option is also being added now!